src/main/java/seedu/address/model/person/Survey.java

```diff
@@ -38,7 +47,7 @@ public static boolean isValidSurvey(String test) {
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof Survey // instanceof handles nulls
-                && survey.equals(((Survey) other).survey)); // state check
+                        && survey.equals(((Survey) other).survey)); // state check
```

Comment:
I think the equals method should include the check if both surveys are marked as done or not done.

Reply:
Thanks for catching this! Should be fixed now